xml/ha_storage_protection.xml

```diff
@@ -348,6 +343,16 @@
        </para>
       </listitem>
      </varlistentry>
+     <varlistentry>
+      <term><literal>stonith-watchdog-timeout</literal> in the CIB</term>
+      <listitem>
+       <para>
+        This timeout is set in the CIB as global cluster property. If not set
+        explicitly, it defaults to <literal>0</literal>, which is appropriate for
+        using SBD with 1-3 devices. For use of SBD in diskless mode, see FIXME
```

Comment:
That FIXME shouldn't be there I guess. :wink:

Reply:
Fixed (replaced by xref).